Calibration/HcalCalibAlgos/plugins/RecAnalyzerMinbias.cc

```diff
@@ -504,8 +499,7 @@ void RecAnalyzerMinbias::analyze(const edm::Event& iEvent, const edm::EventSetup
     h_AmplitudeHF_->Fill(amplitudefullHF);
   }
 
-  edm::Handle<HBHERecHitCollection> hbheMB;
-  iEvent.getByToken(tok_hbherecoMB_, hbheMB);
+  const edm::Handle<HBHERecHitCollection>& hbheMB = iEvent.getHandle(tok_hbherecoMB_);
   if (!hbheMB.isValid()) {
     edm::LogWarning("RecAnalyzer") << "HcalCalibAlgos: Error! can't get hbhe product!";
```

Comment:
```suggestion
    edm::LogWarning("RecAnalyzerMinbias") << "HcalCalibAlgos: Error! can't get hbhe product!";
```
This is the correct module name